src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -60,8 +60,7 @@ public static String formatErrorMsg(String resource) {
             {"R_failedValidate", "failed to validate values in $0} "}, {"R_tableNotDropped", "table not dropped. "},
             {"R_connectionReset", "Connection reset"}, {"R_unknownException", "Unknown exception"},
             {"R_deadConnection", "Dead connection should be invalid"},
-            {"R_wrongExceptionMessage", "Wrong exception message"},
-            {"R_wrongSqlState", "Wrong sql state"},
+            {"R_wrongExceptionMessage", "Wrong exception message"}, {"R_wrongSqlState", "Wrong sql state"},
```

Comment:
We should put one error message on each line

Reply:
I didn't do that manually, the formatter did that